### ISSUE TITLE
The residual connection in Feed Forward block of encoder is incorrect.

### ISSUE THREAD

**Issue by Pravin74** [strutive07/transformer-tensorflow2.0#14]
output, attention = self.multi_head_attention(inputs, inputs, inputs, mask)
output = self.dropout_1(output, training=training)
output = self.layer_norm_1(tf.add(inputs, output))  # residual network
output_temp = output
output = self.position_wise_feed_forward_layer(output)
output = self.dropout_2(output, training=training)
#output = self.layer_norm_2(tf.add(inputs, output))  # residual network (**Incorrect**)
**output = self.layer_norm_2(tf.add(output_temp, output))**  #correct

**Comment by strutive07** [comment]
Thank you for report this error and sorry to check this issue too late.
I fix it on this commit https://github.com/strutive07/transformer-tensorflow2.0/commit/436c4db5121b5d41bf3ca31e012e92adf803dd06
